### DISCUSSION TITLE
Screen recording

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2708] [Q&A] [answered]
Is it feasible to enable screen recording on a per-test-function basis rather than just saving screenshots?

**Comment by mdmintz** [comment, marked as answer]
Use an existing tool with SeleniumBase such as Zalenium for recording videos:
https://opensource.zalando.com/zalenium/
It works like a Selenium Grid.
See https://seleniumbase.io/seleniumbase/utilities/selenium_grid/ReadMe/ for instructions on using SeleniumBase to connect to a Selenium Grid.
